core/src/main/java/org/apache/accumulo/core/client/admin/InstanceOperations.java

```diff
@@ -81,6 +81,15 @@
   List<String> getTabletServers();
 
   /**
+   * List the tablet server status
+   *
+   * @return A list of tablet server status.
+   * @since 2.0.0
+   */
+
+  List<Map<String,String>> getTabletServerStatus() throws AccumuloException;
```

Comment:
From an API design standpoint I would rather see a class returned as that gives more flexibility to evolves the API in the future.  This is a lesson I have learned the hard way.   For example if I wanted to add functionality to get the description of a stat or historical information in the future, that would mean adding more methods to intstance operations.

I would rather see something like the following for the API.

```java
static interfce TabletServerID {
  public String getHost();
  public int getPort();
  public long getSessionId();
}

static interface TabletServersStatus {
  List<TabletServerID> getTabletServers();
  Map<String,String> getTabletServerStatus(TabletServerID tsid);
}

TabletServersStatus getTabletServerStatus();
```